qds_sdk/cloud/oracle_opc_cloud.py

```diff
@@ -0,0 +1,143 @@
+from qds_sdk.cloud.cloud import Cloud
+class OracleOpcCloud(Cloud):
+    '''
+    qds_sdk.cloud.OracleOpcCloud is the class which stores information about oracle opc cloud config settings.
+    The objects of this class can be use to set oracle opc cloud_config settings while create/update/clone a cluster.
+    '''
+
+    def __init__(self):
+        self.compute_config = {}
+        self.network_config = {}
+        self.storage_config = {}
+
```

Comment:
Yes there is no location lonfig, We define region by rest_api_endpoints and that is part of creds.